### ISSUE TITLE
How to use layers class?

### ISSUE THREAD

**Issue by RRHHQ** [deepchem/deepchem#2155]
Hi, I'd like to make graphconvo models composing each layer by myself.
Does anyone know how to use deepchem's layer class?
Sequencial coding like keras is helpful.
like...

 

from keras.models import Sequential
from keras.layers import Dense, Activation

 

model = Sequential([
    Dense(32, input_shape=(784,)),
    Activation('relu'),
    Dense(10),
    Activation('softmax'),
])

I tried to add deepchem layers to keras.models.Sequencial and successed. But I failed to fit the model to training dataset prepared with deepchem featurizer.
Error was like this: "Failed to find data adapter that can handle input: <class "deepchem.data.datasets.DiskDataset>"

**Comment by rbharath** [comment]
Check out our new tutorial on how to make custom models with DeepChem. Hope this is useful! https://github.com/deepchem/deepchem/blob/master/examples/tutorials/Creating%20Models%20With%20TensorFlow%20and%20PyTorch.ipynb

**Comment by arunppsg** [comment]
@rbharath I am not sure that the tutorial https://github.com/deepchem/deepchem/blob/master/examples/tutorials/Creating_Models_with_TensorFlow_and_PyTorch.ipynb is not much useful. We probably need a better tutorial on how to compose and use layers.

**Comment by peastman** [comment]
This tutorial illustrates building up a graphconv model from individual layers: https://github.com/deepchem/deepchem/blob/master/examples/tutorials/Introduction_to_Graph_Convolutions.ipynb
